cms/jinja2/templates/components/streamfield/image_block.html

```diff
@@ -1,4 +1,57 @@
-{% image(value.image, "width-1000", 'loading="lazy"' %}
-{% if value.caption %}
-    <p>{{ value.caption }}</p>
-{% endif %}
+<!--
+    What is considered a large vs small image can vary based on the context of use.
+-->
+{# templates/components/streamfield/image_block.html #}
+{% set large_img = image(value.image, "width-800") %}
+{% set small_img = image(value.image, "width-400") %}
+
+{% from "components/image/_macro.njk" import onsImage %}
+
+{# fmt:off #}
+<div>
+    {# Figure Title #}
+    {% if value.figure_title %}
+        <h3 class="ons-chart__title">{{ value.figure_title }}</h3>
+    {% endif %}
+
+    {# Figure Subtitle #}
+    {% if value.figure_subtitle %}
+        <p>{{ value.figure_subtitle }}</p>
+    {% endif %}
+
+    {# Image #}
+    {{
+        onsImage({
+            "image": {
+                "smallSrc": small_img.url,
+                "largeSrc": large_img.url
+            },
+            "alt": value.image.description,})
+    }}
+
+    {# Source #}
+    {% if value.supporting_text %}
+        <figcaption class="ons-chart__caption">
+            Source - {{ value.supporting_text }}
+        </figcaption>
+    {% endif %}
+
+    {# Notes #}
+    {% if value.notes_section %}
+        <div class="ons-u-mt-s">
+            <h4 class="ons-u-fs-r--b ons-u-mb-2xs">Notes</h4>
+            <p>{{ value.notes_section|richtext }}</p>
+        </div>
+    {% endif %}
+
+    {# Download #}
+    {% if value.download %}
+        <div class="ons-u-mt-s">
+            <h5 class="ons-u-fs-r--b ons-u-mb-2xs">Download {{ value.figure_title or 'image' }}</h5>
+            <a href="{{ large_img.url }}" download>
```

Comment:
I think it would be good to use the `download` attribute to define the filename of the downloaded image, to keep things neat and consistent.

Reply:
Good catch, this is also a requirment on the JIRA card. I seemed to have missed it.

Done: [17bf248](https://github.com/ONSdigital/dis-wagtail/pull/437/commits/17bf248f741b3857a0cfe7ba741aaa00fb217067)

Reply:
@MaciekBaron https://github.com/ONSdigital/dis-wagtail/pull/437/commits/76d71e25adacc97159db7fe73fdc88fdbfea9b6a